### PR TITLE
Simplified version fixing tourism=information issuses #4247 as well

### LINKED ISSUE #4247: Remove rendering for tourism=information without supported value of information=*
### Expected behavior
* The tag `tourism=information` can be used for a wide variety of barely-related features. 
* 93% of features have the key `information=*` which more clearly specifies the feature. 
* The most common values are already supported with specific icons in this style: information=guidepost is most common, followed by information=board,   information=map, information=office. A tourism office is quite different from a guidepost or a public map and should not be confused. 
* Therefore, we should not be showing a symbol for features missing information=* or features with undocumented values of information=*

### Actual behavior
* We are rendering a non-specific “i” icon for all `tourism=information` objects
* This includes misleading uses such as `information=route_marker` and `information=trail_blaze` - the later appears to be a simple cut or paint blotch on a tree which helps mark the location of a rural pathway or trail, though it’s undocumented. See https://taginfo.openstreetmap.org/keys/information#values
* We are also rendering this symbol for the 7% of `tourism=information` objects which have no sub-tag, and could be anything from a guidepost to a visitor centre to a tourism office or a public map. 
 
### Examples with location

https://www.openstreetmap.org/node/7656793394
Screen Shot 2020-11-07 at 12 05 41

Link to image of this feature: https://images.mapillary.com/6kr3SGuwakA3RAxMDut2Ax/thumb-2048.jpg

https://www.openstreetmap.org/node/929360773
Screen Shot 2020-11-07 at 12 07 58

### PR DESCRIPTION
**Pull request by map-per**
Fixes #4247
Simplified version that fixes tourism=information issuses #4247 as well

Changes proposed in this pull request:
Made tourism=information without or with undocumented information=* Tag disapperar

Test rendering with links to the example places:

Test with tourism=information + information=yes: https://www.openstreetmap.org/node/4945458569
Before
OSM (3)

After
Screenshot (653)

Tested information=office, board, guidepost as well and as expekted no changes occured. 
The Names of unwanted points and tourism=information points without information=* don't show up either.

### CODE REVIEW

**Comment by map-per** [comment]
Is this change Ok or do I have to improve something?

**Review by imagico — DISMISSED** [review]
Works fine.

File: project.mml [hunk 1]
```diff
@@ -1524,7 +1524,7 @@ Layer:
                                           OR (boundary = 'protected_area' AND tags->'protect_class' IN ('1','1a','1b','2','3','4','5','6'))
                                           THEN boundary END,
                 'leisure_' || CASE WHEN leisure IN ('nature_reserve') THEN leisure END,
-                'tourism_' || CASE WHEN tourism IN ('information') THEN tourism END,
+                'tourism_' || CASE WHEN tourism IN ('information') AND tags->'information' IN ('audioguide', 'board', 'guidepost', 'office', 'map', 'terminal') THEN tourism END,
```
Comment (imagico) [inline project.mml:1527]: tactile_map is missing in the list of values here.

**Review by imagico — CHANGES_REQUESTED** [review]
Works as expected, just missing a value in SQL.  If you can add that this would be great.  Sorry for the delay in reviewing.

File: project.mml (hunk 1 above)
Comment (map-per) [reply project.mml]: Thanks, corrected the mistake and tested

**Review by imagico — APPROVED** [review]
Looks good now, thanks.

**Comment by map-per** [comment]
Sorry, requesting review again as accidental.

**Pull request merged**